ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutor.java

```diff
@@ -115,21 +129,75 @@ private void showWelcomeMessage() {
   }
 
   private void executeStatements(final String queries) {
-    final List<QueryMetadata> queryMetadataList = ksqlEngine.createQueries(queries, ksqlConfig);
-    for (QueryMetadata queryMetadata : queryMetadataList) {
-      if (queryMetadata instanceof PersistentQueryMetadata) {
-        PersistentQueryMetadata persistentQueryMetadata = (PersistentQueryMetadata) queryMetadata;
-        persistentQueryMetadata.start();
+    final List<Pair<String, Statement>> statementPairs =
+        ksqlEngine.parseStatements(queries, ksqlEngine.getMetaStore().clone());
+    final Map<String, Object> configProperties = new HashMap<>();
+    for (Pair<String, Statement> statementPair: statementPairs) {
+      final String statementString = statementPair.getLeft();
+      final Statement statement = statementPair.getRight();
+      if (statement instanceof SetProperty) {
```

Comment:
Can we get rid of all the `instanceof` here and in other methods. Use a `Map<Class<? extends Statement>, Function>` and then just do a get from the map and dispatch. Code becomes much more readable

Reply:
Using a map as you suggested.